libs/context-api/token/entity-workspace-context.token.ts

```diff
@@ -0,0 +1,7 @@
+import { UmbEntityWorkspaceContextInterface } from '../../../src/backoffice/shared/components/workspace/workspace-context/workspace-entity-context.interface';
```

Comment:
We should avoid loading from the src folder in a lib. We should move this file to the workspace lib instead.

Reply:
Ah, good one! Thanks. will correct right away